# A common message type to indicate the outcome of a service call.
uint8 exception_type
string message
